src/store.js

```diff
@@ -0,0 +1,81 @@
+export const TOGGLE = 'action/toggle';
+export const LOAD_TODOS = 'action/load_todos';
+export const SWAP_TODOS = 'action/swap';
+export const ADD_TODO = 'action/add_todo';
+export const EDIT_TODO = 'action/edit_description';
+export const DELETE_TODO = 'action/delete_todo';
+export const CLEAR_COMPLETED = 'action/clear_completed';
```

Comment:
You can try exporting the whole class with static methods and avoid multiple exports. So you can easily access all your functionality and save some code lines.

Reply:
Thanks for your positive review. Can you please explain the syntax and how I can implement this requirement?

Reply:
Hi Usama, sure...

Instead of exporting each function individually, you can export a whole class with all its functionality:

```javascript
export default class SomeClass {
  static function1(){
    return 1;
  }

  static function2(){
    return 2;
  }
}
```

Then you can simply import it in your other files and use it like this:

```javascript
import SomeClass from './someClass.js';
  console.log(SomeClass.function1()); // Expected output: 1
  console.log(SomeClass.function2()); // Expected output: 2
}
```

Reply:
Done. Thanks for your time ❤️